environment.yml

```
@@ -1,6 +1,8 @@
 channels:
-  - manics
+  - conda-forge
 dependencies:
+  - jupyter-server-proxy>=1.4
+  - pip
   - websockify
   - pip:
-    - .
\ No newline at end of file
+      - jupyter-desktop-server==0.1.3
```

```suggestion
      - .
```

I think this should still be '.', so it's running the latest version in the repo.

---

I had to update the Dockerfile as well, see the last commit